workflow-templates/dependabot/workflow-template-copies/.github/workflows/release-go-task.yml

```diff
@@ -149,11 +160,9 @@ jobs:
           tar -czvf "$PACKAGE_FILENAME" \
           -C "${{ env.PROJECT_NAME }}_osx_${{ matrix.artifact.name }}/" "${{ env.PROJECT_NAME }}" \
           -C ../../ LICENSE.txt
-          CHECKSUM_LINE="$(shasum -a 256 $PACKAGE_FILENAME)"
           echo "PACKAGE_FILENAME=$PACKAGE_FILENAME" >> $GITHUB_ENV
-          echo "::set-output name=checksum-${{ matrix.artifact.name }}::$CHECKSUM_LINE"
```

Comment:
@MatteoPologruto In this PR we forgot to remove https://github.com/arduino/tooling-project-assets/pull/277/files#diff-3377c3cd39d3d2deebfce56bb1ffc4097c576a1eb06511dc4907daa926987627L56-L58